Docs build is failing

As of [this commit](https://github.com/moj-analytical-services/splink/tree/2c3ea240f91ed01bd884d53304efba3621c71e54) (though probably unrelated) the docs deploy step is failing (see e.g. [this run](https://github.com/moj-analytical-services/splink/actions/runs/6880110600/job/18713608048)).

Error:
```
ERROR    -  Config value 'plugins': Plugin 'mkdocstrings' option 'custom_templates': The path '/home/runner/work/splink/splink/templates' isn't an existing directory.
Aborted with 1 Configuration Errors!
Error: Process completed with exit code 1.
```

Possibly a change in plugin version? But haven't looked into at all

--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -52,7 +52,6 @@ plugins:
         python:
           rendering:
             show_source: false
-      custom_templates: templates
   - git-revision-date-localized:
       enable_creation_date: true
       type: timeago

